app/utils/createPage.js

```diff
@@ -15,8 +16,8 @@ const createPage = ({
   requireFeatures = [],
   requireLogin = true,
   LoaderComponent = () => <Loader />,
-  //TODO(naman): add default error layout
-  ErrorComponent = () => 'There was some error.',
+  //TODO(naman): add default error layout - Done!
```

Comment:
A todo comment is removed when completed.

Reply:
Done